docs/BUILD.bazel

```
@@ -140,71 +139,12 @@ sphinx_build_binary(
     ],
 )
 
-_REQUIREMENTS_TARGET_COMPATIBLE_WITH = select({
-    "@platforms//os:linux": [],
-    "@platforms//os:macos": [],
-    "@platforms//os:windows": [],
-    "//conditions:default": ["@platforms//:incompatible"],
-}) if BZLMOD_ENABLED else ["@platforms//:incompatible"]
-
-# Run bazel run //docs:requirements.update
-genrule(
-    name = "requirements",
+# Run bazel run //docs:requirements_update
+pin(
+    name = "requirements_update",
```

```suggestion
    name = "requirements.update",
```

I'd like to retain the `:requirements` and `requirements.update` names as part of the API. It's already established by pip_compile(), deviating from that just seems needless, and retaining it makes switching easier.

I realize this doesn't define a `:requirements` target, which is fine for now, but we can at least keep the `:requirements.update` name.